src/mod/server.mod/servmsg.c

```diff
@@ -1343,6 +1396,14 @@ static int got410(char *from, char *msg) {
   return 1;
 }
 
+/* got417: ERR_INPUTTOOLONG. Client sent a message longer than allowed limit */
+static int got417(char *from, char *msg) {
+  newsplit(&msg);
+  putlog (LOG_SERV, "*", "MESSAGE-TAG: %s reported error: %s", from, msg);
```

Comment:
this isn't message tag specific though, is it?

Reply:
417 was introduced as part of the msg-tags spec, not part of the base rfc. "Servers MUST reply with the ERR_INPUTTOOLONG (417) error numeric if a client sends a message with more tag data than the allowed limit. " So I read this as specific to tags, not the line. Thoughts?